server/controllers/users.js

```diff
@@ -0,0 +1,48 @@
+import bcrypt from 'bcrypt';
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import db from '../models';
+
+dotenv.load();
+const secret = process.env.SUPER_KEY;
+const users = db.user;
+const saltRounds = 10;
+
+export default {
+  signup(req, res) {
+    return users
+      .create({
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        userName: req.body.userName,
+        email: req.body.email,
+        password: bcrypt.hashSync(req.body.password, saltRounds)
```

Comment:
just pass 10 as a parameter instead of assigning it to a `saltRounds` variable first